backend/user-service/src/routes/user.routes.ts

```diff
@@ -1,8 +1,10 @@
+import { handleCreateUser, handleUpdateProfile } from '../controllers/user.controller'
+
 import { Router } from 'express'
-import { handleCreateUser } from '../controllers/user.controller'
 
 const router = Router()
 
-router.post('/', handleCreateUser)
+router.post('/create', handleCreateUser)
+router.post('/update-profile', handleUpdateProfile)
```

Comment:
The update API should also have an id param in the route that will identify which resource is being updated

Reply:
yup correct, was using the id provided in the post request. since we decided to use put request, ill add it in